src/gui/tray/usermodel.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "accountmanager.h"
 #include "owncloudgui.h"
-#include <pushnotifications.h>
+#include "pushnotifications.h"
```

Comment:
I wouldn't change that, `pushnotifications.h` is in `libsync`

Reply:
Sorry, reverted